website/docs/error-reference.md

```diff
@@ -388,6 +388,20 @@ See [5014](#5014). 5036 is the same error as [5014](#5014) but slightly modified
 to allow more common Ruby idioms to pass by in `# typed: true` (5036 is only
 reported in `# typed: strict`).
 
+## 5041
+
+Sorbet does not allow inheriting from a class which inherits from `T::Struct`.
+
+```ruby
+class S < T::Struct
+  prop :foo, Integer
+end
+
+class Bad < S; end # error
+```
+
+In the future, this error may be subsumed by [5047](#5047).
```

Comment:
I think it's not useful to say "may be subsumed by 5047".

Reply:
My thought was that eventually we might change classes that inherit from `T::Struct` to just be final instead of having the one-off logic in `definition_validator`, and if such a time were to come then this error code might go away. Is this the eventual plan? If yes, is it useful to say that here, or do we not need to bother?

Reply:
Our users aren’t memorizing our error codes. They care about how to fix the error they’re seeing, not how many similar error codes there are